core/src/main/java/com/example/AutomatedReviewTest.java

```diff
@@ -0,0 +1,53 @@
+package com.example;
+
+import java.util.*;
+import java.util.logging.*;
+
+/**
+ * The goal of this code is to find out if code analysis will automatically add
+ * commentary to a pull request when it finds problems with the code.
+ */
+public final class AutomatedReviewTest {
+    
+    private Logger LOGGER = Logger.getLogger(AutomatedReviewTest.class.getName());
+    
+    protected double one   = 1.0d;
+    protected double two   = 2.0d;
+    protected double three = 5.0d;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Assign this magic number 5.0d to a well-named constant, and use the constant instead. ([squid:S109](https://sonarcloud.io/organizations/coderanch-corral/rules#rule_key=squid%3AS109))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.coderanch%3Ablackjack&pullRequest=9&issues=AWZEEZasvOM6zLh9bdFl&open=AWZEEZasvOM6zLh9bdFl)

[ISSUE_KEY]: AWZEEZasvOM6zLh9bdFl